src/Illuminate/Routing/Router.php

```diff
@@ -1213,7 +1214,17 @@ protected function cleanFilterParameters(array $parameters)
 	 */
 	public function prepareResponse($request, $response)
 	{
-		if ( ! $response instanceof SymfonyResponse)
+		if ($response instanceof PsrResponseInterface)
+		{
+			$psrResponse = $response;
+			$response = new Response(
+			  $psrResponse->getBody()->__toString(),
```

Comment:
I think asting to string gives problem when It throws an Exception.
Shoud I just indent the parameters with tabs also?